strawberryfields/api_client.py

```diff
@@ -141,37 +141,36 @@ class APIClient:
 
     USER_AGENT = "strawberryfields-api-client/0.1"
 
-    ALLOWED_HOSTNAMES = ["localhost", "localhost:8080", "platform.strawberryfields.ai"]
+    ALLOWED_HOSTNAMES = [
+        "localhost",
+        "localhost:8080",
+        "platform.strawberryfields.ai",
+    ]
 
     DEFAULT_HOSTNAME = "localhost"
 
-    ENV_KEY_PREFIX = "SF_API_"
-    ENV_AUTHENTICATION_TOKEN_KEY = "{}AUTHENTICATION_TOKEN".format(ENV_KEY_PREFIX)
-    ENV_API_HOSTNAME_KEY = "{}API_HOSTNAME".format(ENV_KEY_PREFIX)
-    ENV_USE_SSL_KEY = "{}USE_SSL".format(ENV_KEY_PREFIX)
```

Comment:
Were these not used anywhere else?

Reply:
No they are not. These were left behind after a previous refactor.